src/deepak/AssignmentSeven.java

```diff
@@ -0,0 +1,71 @@
+package deepak;
+
+public class AssignmentSeven {
+	
+	void evenNumbers(int start, int end) {
+		
+		System.out.println("Printing Even numbers in given range"+" "+ start +" "+ "TO"+" "+ end);
+		
+		for(int i=start; i<=end; i++) {
+			
+			if(i%2==0) {
+				
+				System.out.println(i);
+			}
+		}
+		
+	}
+	
+	void numbersDivisibleByFive(int start, int end) {
+		
+		System.out.println("Printing numbers divisible by five only in given range"+" "+ start +" "+ "TO"+" "+ end);
+		
+		for(int i=start; i<=end; i++) {
+			
+			if(i%5==0) {
+				
+				System.out.println(i);
+			}
+		}
+		
+	}
+	
+	void numbersDivisibleByFiveAndThree(int start, int end) {
+		
+		System.out.println("Printing numbers divisible by five and three in given range"+" "+ start +" "+ "TO"+" "+ end);
+		
+		for(int i=start; i<=end; i++) {
+			
+			if(i%5==0 && i%3==0) {
+				
+				System.out.println(i);
+			}
+		}
+		
+	}
+	
+	void numbersDivisibleBySevernOrThirteen(int start, int end) {
+		
+		System.out.println("Printing numbers divisible by seven OR thirteen in given range"+" "+ start +" "+ "TO"+" "+ end);
+		
+		for(int i=start; i<=end; i++) {
+			
+			if(i%7==0 || i%13==0) {
+				
```

Comment:
this condition not fulfill below

 7 is divisible by 7
		 13 is divisible by 13
		 14 is divisible by 7
		 21 is divisible by 7
		 26 is divisible by 13
		 28 is divisible by 7
		 35 is divisible by 7
		 39 is divisible by 13
Requirement 

You should write in if else condition.
Next be careful.